tripy/tripy/frontend/module/layernorm.py

```diff
@@ -47,11 +47,12 @@ class LayerNorm(Module):
     eps: float
     """A value added to the denominator to prevent division by zero."""
 
-    def __init__(self, normalized_shape: int, dtype: datatype.dtype = datatype.float32) -> None:
+    def __init__(self, normalized_shape: int, dtype: datatype.dtype = datatype.float32, eps: float = 1e-5) -> None:
         """
         Args:
             normalized_shape: The size of the feature dimension of the input over which normalization is performed.
             dtype: The data type to use for the weight and bias parameters.
+            eps: :math:\epsilon\ value 
```

Comment:
Do we need the trailing slash?
```suggestion
            eps: :math:\epsilon value 
```